src/Symfony/Component/Serializer/Normalizer/MimeMessageNormalizer.php

```diff
@@ -42,7 +42,18 @@ public function __construct(PropertyNormalizer $normalizer)
         $this->headersProperty = new \ReflectionProperty(Headers::class, 'headers');
     }
 
-    public function setSerializer(SerializerInterface $serializer): void
+    public function getSupportedTypes(?string $format): array
+    {
+        return [
+            Message::class => true,
+            Headers::class => true,
+            HeaderInterface::class => true,
+            Address::class => true,
+            AbstractPart::class => true,
```

Comment:
`__CLASS__ === static::class`

Reply:
The class is final so true is enough right?

Reply:
indeed